sparkstreaming-job/src/main/java/zipkin/sparkstreaming/job/ZipkinSparkStreamingProperties.java

```diff
@@ -13,41 +13,42 @@
  */
 package zipkin.sparkstreaming.job;
 
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import zipkin.sparkstreaming.SparkStreamingJob;
 
 @ConfigurationProperties("zipkin.sparkstreaming")
 public class ZipkinSparkStreamingProperties {
-  String sparkMaster;
-  List<String> sparkJars;
-  Map<String, String> sparkProperties;
+  String master;
```

Comment:
I think the only it might make sense to do would be the properties (possibly the list), the non-collection fields would be unsettable if we made them final. cc @shakuzen